mne/io/eeglab/eeglab.py

```diff
@@ -525,8 +525,13 @@ def _read_eeglab_events(eeg, event_id=None, event_id_func='strip_to_integer'):
     if event_id is None:
         event_id = dict()
 
-    types = [event.type for event in eeg.event]
-    latencies = [event.latency for event in eeg.event]
+    try:
+        types = [event.type for event in eeg.event]
+        latencies = [event.latency for event in eeg.event]
+    except TypeError:
```

Comment:
Instead of `try/except` (which we generally try to avoid) can you instead check to see if it's dim > 0?